docs/cli/pyats_undevelop.rst

```diff
@@ -0,0 +1,72 @@
+pyats undevelop
+===============
+
+Almost all pyATS package repositories come with a `make undevelop` command 
+which removes a package from development mode, effectively uninstalling it. 
+While this command is useful, it doesn't go the extra mile of reinstalling 
+the uninstalled packages afterwards, often resulting in a non-funtional 
+pyATS installation. This is where the `pyats undevelop` command comes in.
+
+The `pyats undevelop` helper command is designed to ease and expedite 
+the process of removing pyATS packages from development mode. It checks that 
+all of your pyATS packages are up-to-date, removes specified packages from 
+development mode, and then reinstalls the most up-to-date version of those 
+packages using pip. No need to run each of these commands manually anymore.
+
+.. code-block:: text
+
+    Usage:
+      pyats undevelop [packages...] [options]
+    
+    Usage Examples:
+      pyats undevelop all
+      pyats undevelop pyats --skip-version-check
+      pyats undevelop genie.libs.parser genie.trafficgen
+      pyats undevelop genie.libs unicon.plugins
+    
+    Description:
+      Removes listed pyATS packages from development mode. Each listed package is 
+      removed from development mode with 'make undevelop' and then is reinstalled 
+      using 'pip install <package>'. Internal Cisco packages will be reinstalled if 
+      the pyATS installation is internal, otherwise external packages will be 
+      reinstalled instead.
+    
+    Undevelop Options:
+      packages              Packages to put into development mode. Available choices: all, cisco-
+                            distutils, genie, genie.libs, genie.libs.clean, genie.libs.conf,
+                            genie.libs.filetransferutils, genie.libs.health, genie.libs.ops,
+                            genie.libs.robot, genie.libs.sdk, genie.libs.parser, genie.telemetry,
+                            genie.trafficgen, pyats, pyats.contrib, rest.connector, unicon,
+                            unicon.plugins
```

Comment:
Same thing here, though admittedly I'm thinking it might not be a great idea now 🤔